src/sbt-test/semver/tagging/test

```diff
@@ -6,7 +6,7 @@
 $ exec git init
 $ exec git add .
 $ exec git commit -am 'Initial commit.'
-$ exec git tag v0.0.1
+$ exec git tag v1.0.1
```

Comment:
Do you have another test like this that does tagging at v0? I wouldn't want to lose coverage.

Reply:
@htmldoug should we add this test back to the plugin? If so, that should go to the internal plugin version.